runtimes/common/src/bonded_currencies/mod.rs

```diff
@@ -65,12 +65,12 @@ impl<Target: Get<L>, L: PartialEq + Eq> Convert<L, Either<(), L>> for TargetFrom
 
 /// Wrapper struct for [UnionOf] to implement the [metadata::Inspect] trait,
 /// needed for the pallet_bonded_coins module.
-pub struct WrapperNativeAndForeignAssets<Left, Right, Criterion, AssetKind, AccountId>(
+pub struct NativeAndForeignAssets<Left, Right, Criterion, AssetKind, AccountId>(
```

Comment:
Generics `Left` and `Right` should rather be `Native` and `ForeignAssets`, I would suggest.

Reply:
Renamed in: [37299a7](https://github.com/KILTprotocol/kilt-node/pull/818/commits/37299a78ddb518cae0ae5a5450bd6a1145ec444c)